### ISSUE TITLE
[CAIA A11y Research - PILOT] Disability Benefits - Pilot Session - Employee Experience 21-526EZ, Max CFI (Jan. 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#74019]
# CAIA A11y Support

This is a child ticket of the research ticket [69999](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69999) - created to track work completed for the pilot session. 

## Assignee
- Jamie

## Description

A CAIA a11y will participate as in a pilot of the Disability Benefit team's [21-526EZ, Max CFI research study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69999).

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` on `desktop` and `screen magnifier` on `desktop` would behave.

### Assistive Tech Options

- **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
- **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Team Info.

- Team name: Disability benefits employee experience
- OCTO-DE product owner: @jenniferbertsch
- Product manager: @amylai-nava
- Designer: @carodew @KimLadin @ChristineRoseSteiffer
- Accessibility specialist:
- Content writer: @carodew
- Slack channel: [#benefits-employee-experience-design](https://dsva.slack.com/archives/C03C8661PAP)

## Pilot Session
- Monday, Jan. 24, 2024 at 3:30 p.m. 
- Prototype in [Codepen](https://codepen.io/team/a6it/live/KKEMGGE)
- Research [Convo Guide Link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/abd-vro/2024-01-Max-CFI-Refinements/2024-01-Max-CFI-Refinements-Conversation-Guide.md)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] Determine date and time: 1/24/24, 3:30 p.m.
- [x] Add to a11y Capacity Calendar
- [x] Schedule CAIA a11y: @coforma-jamie
- [x] Review research [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/abd-vro/2024-01-Max-CFI-Refinements/2024-01-Max-CFI-Refinements-Conversation-Guide.md)
- [ ] Review [prototype in CodePen](https://codepen.io/team/a6it/live/KKEMGGE)
- [ ] Run pilot
- [ ] Share findings: View a Recent [Example from Jamie](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/MHV%20Cartography%20Team/2024-01-18-Design-Review-Cartography-Team-New-Navigation.md)
```

**Comment by sara-amanda** [comment]
Hi, @ChristineRoseSteiffer, @coforma-jamie will be assisting with your pilot next week re: Slack. This is the ticket CAIA is using to organize the pilot-related information. I will also tag Jamie in the Slack chat. 😄 We are so happy you are testing with AT users! 

> Christine Rose Steiffer
>   [1 day ago](https://dsva.slack.com/archives/C8E985R32/p1706033135103599?thread_ts=1705512765.800649&cid=C8E985R32)
> Also, I would love to have a brief meeting with a CAIA support person who might be in the sessions with us next week. This is our first time testing with AT users so I would love to talk about how they typically go and how we can best prepare for next week! Thanks so much

**Comment by sara-amanda** [comment]
## Closed Ticket Notes
- [x] Adding closed date to GHP Sitewide field: `1/25/2024`
- [x] Added note about notes because `notes were optional` The research team took notes, due to CAIA participating in the pilot with one team member. 
- [x] Marking off all tasks as completed.

_Thank you for testing your pilot with Jamie, Christine! We hope your study goes well and that this was helpful._

cc: @coforma-terry
